packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1739,18 +1739,18 @@ describe('TransactionController', () => {
   });
 
   describe('initApprovals', () => {
+    const txMeta = {
+      from: ACCOUNT_MOCK,
+      hash: '1337',
+      id: 'mocked',
+      chainId: toHex(5),
+      status: TransactionStatus.unapproved,
+    };
```

Comment:
Better typing that lets us remove the `txMeta as any` assertions.

```suggestion
    const txMeta: TransactionMeta = {
      hash: '1337',
      id: 'mocked',
      chainId: toHex(5),
      status: TransactionStatus.unapproved,
      time: 0,
      txParams: { from: ACCOUNT_MOCK } as TransactionParams,
    }
```

Reply:
Requires import statement update:

```diff
@@ -30,7 +30,11 @@ import type {
   TransactionConfig,
 } from './TransactionController';
 import { TransactionController } from './TransactionController';
-import type { TransactionMeta, DappSuggestedGasFees } from './types';
+import type {
+  TransactionMeta,
+  DappSuggestedGasFees,
+  TransactionParams,
+} from './types';
 import { WalletDevice, TransactionStatus, TransactionType } from './types';
 import { estimateGas, updateGas } from './utils/gas';
 import { updateGasFees } from './utils/gas-fees';
```